tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -229,6 +231,30 @@ Object.Pipeline {
 			Object.Widget.host-copier.1 {
 				stream_name	'$HEADSET_CAPTURE_PCM_STREAM_NAME'
 				pcm_id $HEADSET_PCM_ID
+				IncludeByKey.SSP_HS_FMT_24 {
```

Comment:
This looks wrong. Why would the SSP format be reflected in the host-copier stuff?

Reply:
@plbossart, you are right. It shouldn't be in the host-copier, but dai-copier specific. Besides, the host capture has supported those formats. I will remove this part. Thanks!